subsys/caf/modules/sensor_manager.c

```diff
@@ -180,7 +180,11 @@ static void sensor_wake_up_post(const struct sm_sensor_config *sc, struct sensor
 
 static void trigger_handler(const struct device *dev, const struct sensor_trigger *trigger)
 {
-	sensor_trigger_set(dev, trigger, NULL);
+	int ret = sensor_trigger_set(dev, trigger, NULL);
```

Comment:
let's try to not mix names - in other function (line 213) this variable is called `err`

Reply:
Add n other places we use ret, see 451 and 485 :) 
But as you wish I change my ret/err